example/index.html

```diff
@@ -0,0 +1,148 @@
+console.log(galacticConversionResult) // { isSuccess: true, value: '0C55:00D5:0922:0234', errorMessage: '' }</code></pre>
+        </section>
+
+        <section style="margin-bottom: 4em">
+            <h2 class="noselect">Using the converters</h2>
+            <p class="noselect">Once the library has been imported using one of the methods described above, you will be
+                able to use the functions the library provides.</p>
+            <pre><code class="language-javascript">const galacticConversionResult = PortalCode({ code: '023456123456' }).toGalacticCoordinates();
+if (galacticConversionResult.isSuccess === true) {
+    console.log(galacticConversionResult.value); // 0C55:00D5:0922:0234
+}
+else {
+    console.warn(galacticConversionResult.errorMessage); // e.g. "Invalid portal code provided"
+}</code></pre>
+        </section>
+
+        <section id="examples">
+            <h2 class="noselect">Example conversions</h2>
+
+            <div class="grid conversion-examples" style="margin-top: 2em">
+                <fieldset>
+                    <label>Portal coordinates</label>
+                    <input id="portal" type="text" name="portal" placeholder="023456123456" oninput="onPortalCode()"
+                        value="023456123456" />
+                    <small class="help-text"><i>Valid characters are 0123456789ABCDEF.</i></small>
+                </fieldset>
+                <fieldset>
+                    <label>Galactic coordinates</label>
+                    <article id="portaltoGalacticCoordinates">0C55:00D5:0922:0234</article>
+
+                    <label>Voxel coordinates</label>
+                    <pre id="portaltoVoxel" style="padding: 1em">{}</pre>
+                </fieldset>
+            </div>
+
+            <hr class="hidden-in-desktop" />
+
+            <div class="grid conversion-examples" style="margin-top: 2em">
+                <fieldset>
+                    <label>Galactic coordinates</label>
+                    <input id="galactic" type="text" name="galactic" placeholder="0A32:00D5:0F44:0234"
+                        oninput="onGalacticCoord()" value="0A32:00D5:0F44:0234" />
+                    <small class="help-text"><i>Valid characters are 0123456789ABCDEF with semicolons.</i></small>
+                </fieldset>
+                <fieldset>
+                    <label>Portal coordinates</label>
+                    <article id="galactictoGlyph">023456123456</article>
+
+                    <label>Voxel coordinates</label>
+                    <pre id="galactictoVoxel" style="padding: 1em">{}</pre>
+                </fieldset>
+            </div>
+
+            <hr class="hidden-in-desktop" />
+
+            <div class="grid conversion-examples" style="margin-top: 2em">
+                <fieldset>
+                    <label>Voxel coordinates</label>
+                    <textarea id="voxel" type="text" name="voxel" oninput="onVoxelCoord()" placeholder="tbd"
```

Comment:
IMO this textarea should not be resizable

Reply:
true true